gammapy/estimators/flux.py

```diff
@@ -113,6 +113,14 @@ def get_scale_model(self, models):
         ref_model = models[self.source].spectral_model
         scale_model = ScaleSpectralModel(ref_model)
 
+        is_free_norm = np.array(
+            [par.is_norm for par in ref_model.parameters.free_parameters]
+        )
+        if np.sum(is_free_norm) > 1:
+            raise ValueError(
+                "Spectral model used can only have a single free norm type parameter."
+            )
+
```

Comment:
There is always only one norm (and no other parameters) in the scale model so this in not necessary

Reply:
I don't think so... Have a look on the tests;)

Reply:
because `is_free_norm` test  the ref_model  but it will be replace by
the scale model in the estimator so the will issue disappears

Reply:
The tests work... how can you explain that, then?